scroll-animations-1/Overview.bs

```diff
@@ -353,18 +353,99 @@ spec: cssom-view-1; type: dfn;
 
 	Values are as defined for ''scroll()''.
 
+	The [=used value=] of 'scroll-timeline-axis'
+	is the axis of the timeline's [=attachment=],
+	or the initial value if there is no [=attachment=].
+
+### Attachment of a Scroll Progress Timeline: the 'scroll-timeline-attachment' property ### {#scroll-timeline-attachment}
+
+	<pre class='propdef'>
+	Name: scroll-timeline-attachment
+	Value: [ local | defer | ancestor ]#
+	Initial: local
+	Applies to: all elements
+	Inherited: no
+	Computed value: a list of the keywords specified
+	Animation type: not animatable
+	</pre>
+
+	Specifies attachment behavior for each [=scroll progress timeline=]
+	associated with this element.
+
+	A scroll timeline has an <dfn>attachment</dfn>,
+	that determines which [=scroll container=] and {{ScrollTimeline/axis}} to use
+	when calculating the [progress of the timeline](#scroll-timeline-progress).
+
+	<dl>
+		<dt><dfn>'local'</dfn>
+		<dd>
+			The [=attachment=] is this timeline.
+
+		<dt><dfn>'defer'</dfn>
+		<dd>
+			The [=attachment=] is the timeline among the flat-tree descendants
+			that satisfies all of the following:
+			 - The timeline has a matching 'scroll-timeline-name'.
+			 - The timeline has [=ancestor=] attachment.
+			 - The timeline has no intermediate [=defer|deferred=] timeline
+			   that would capture it.
+
+			If more than one such candidate exists,
+			then this timeline has no [=attachment=].
+
+		<dt><dfn>'ancestor'</dfn>
+		<dd>
+			The timeline has no [=attachment=].
+
+			Timelines with [=ancestor=] attachment are treated as nameless
+			for the purposes of timeline lookup;
+			they can't be referenced by 'animation-timeline'.
+	</dl>
+
+	Timelines that have no [=attachment=] are [=inactive timeline|inactive=].
+
+	<div class=example>
+		Using a [=defer|deferred timeline=],
+		an element can effectively refer to timelines
+		that appear <em>after it</em> in tree order.
+		For example, the following creates an animation on <code>#target</code> that
+		is ultimately linked to a scroll timeline defined on the subsequent sibling.
+
+		<pre class=lang-css>
+		@keyframes anim {
+		  from { color: red; }
+		  to { color: green; }
+		}
+
+		#root {
+		  scroll-timeline-name: mytimeline;
+		  scroll-timeline-attachment: defer;
+		}
+
+		#root #target {
+		  animation: anim auto;
```

Comment:
```suggestion
		  animation: anim;
```

We're still discussing whether `auto` is a valid keyword inside `animation`, but it's definitely the initial value for `animation-duration` so leaving it off will work.